pkg/api/tag.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+)
+
+func (s *server) getANewTag(w http.ResponseWriter, r *http.Request) {
+	tagName := mux.Vars(r)["name"]
+	if tagName == "" {
+		s.Logger.Debugf("bzz-tag: invalid tag name : %s", tagName)
+		s.Logger.Error("bzz-tag: invalid tag name")
+		jsonhttp.BadRequest(w, "invalid tag name")
+		return
+	}
+
+	tag, err := s.Tags.Create(tagName, 1, false)
```

Comment:
Should here be total set to 0, as no chunks are uploaded still? And when the chunk is uploaded to increment Total state in Tag.